### ISSUE TITLE
Maximum retries exceeded

### ISSUE THREAD

**Issue by jacob-02** [anuraghazra/github-readme-stats#1094]
Maximum retries exceeded 
Please add an env variable called PAT_1 with your github token in vercel

Show the number of stars and my top languages

![Screenshot from 2021-05-30 20-26-50](https://user-images.githubusercontent.com/74947682/120109082-87e9f500-c185-11eb-8877-6e6d30c2e452.png)

**Comment by Zolyn** [comment]
Same problem.

**Comment by ganler** [comment]
Same problem as well.

**Comment by Madman10K** [comment]
Same for me as well

**Comment by YHPeter** [comment]
Same issue

**Comment by blogSoul** [comment]
Same issue

![image](https://user-images.githubusercontent.com/42922298/120109185-51bb6e00-c1a3-11eb-86a0-9eb313425c6e.png)

**Comment by sobankhanlodhi** [comment]
Guys follow the tutorial properly from here: https://www.youtube.com/watch?v=n6d4KHSKqGk&t=107s

**Comment by blogSoul** [comment]
@sobankhanlodhi  thank you! i solve this problem:)

**Comment by jacob-02** [comment]
thanks! helped a lot!

**Comment by sobankhanlodhi** [comment]
> @sobankhanlodhi thank you! i solve this problem:)

Glad to help.

**Comment by akshattrivedi** [comment]
It was working previously, same problem occurred.

**Comment by tjysdsg** [comment]
tldr:
deploy this repo on vercel.com and use the deployed site instead of github-readme-stats.vercel.app

**Comment by AnthonyZet** [comment]
> Guys follow the tutorial properly from here: https://www.youtube.com/watch?v=n6d4KHSKqGk&t=107s
3:48 After I´ve clicked on deploy in Vercel I´ve got this error:
The value for maxDuration must be between 1 second and 10 seconds, in order to increase this limit upgrade your plan. Any ideas?
![image](https://user-images.githubusercontent.com/67970914/206474598-24a943a6-3785-4072-a8d4-801ec2ea1906.png)

**Comment by rickstaa** [comment]
> > Guys follow the tutorial properly from here: https://www.youtube.com/watch?v=n6d4KHSKqGk&t=107s
> > 3:48 After I´ve clicked on deploy in Vercel I´ve got this error:
> > The value for maxDuration must be between 1 second and 10 seconds, in order to increase this limit upgrade your plan. Any ideas?
> > ![image](https://user-images.githubusercontent.com/67970914/206474598-24a943a6-3785-4072-a8d4-801ec2ea1906.png)

@AnthonyZet I just answered your question in https://github.com/anuraghazra/github-readme-stats/discussions/2318 hope that helps. 👍
